config/locales/ja.yml

```diff
@@ -14,7 +14,7 @@ ja:
   label_name: タイトル
   label_baselayer: 背景レイヤー
   label_nearby: "近く(緯度,経度)"
-  label_layer: "レイヤー種別"
+  label_layer: "レイssssssssssssssssssssssssssssヤー種別"
```

Comment:
@sanak What is this?

Reply:
@dkastl I find your commit has this bug. I send new PR.

Reply:
@smellman , thanks for findig this typo. I didn't see this when merging the translation from Weblate. I merged your PR and it seems to be included now in this PR as well.